editor/src/components/canvas/controls/new-canvas-controls.tsx

```diff
@@ -173,55 +179,67 @@ export const NewCanvasControls = React.memo((props: NewCanvasControlsProps) => {
     [drop],
   )
 
+  const ref = React.useRef<HTMLDivElement | null>(null)
+
+  const elementsOutsideVisibleAreaIndicators = useElementsOutsideVisibleArea(
```

Comment:
I think this should be moved inside the `ElementsOutsideVisibleAreaIndicators`

Reply:
https://github.com/concrete-utopia/utopia/pull/3842/commits/30a89e064c08ac48c0a1ebb9b506cdd91e68d927